commune_app/migrations/0002_add_user_test_data.py

```diff
@@ -0,0 +1,26 @@
+from django.db import migrations, transaction
+
+
```

Comment:
This file is already in #52

Reply:
@kobihk 
how i can delete it but it still work?

Reply:
@ofirtzabari yes because PR #52 still not merged